src/test/java/de/tum/in/www1/artemis/localvcci/LocalCIResourceIntegrationTest.java

```diff
@@ -198,57 +203,12 @@ void testCancelAllRunningBuildJobs() throws Exception {
         request.delete("/api/admin/cancel-all-running-jobs", HttpStatus.NO_CONTENT);
     }
 
-    @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
-    void testCancelBuildJobForCourse() throws Exception {
-        LocalCIBuildJobQueueItem buildJob = processingJobs.get(1L);
-        request.delete("/api/courses/" + course.getId() + "/cancel-job/" + buildJob.id(), HttpStatus.NO_CONTENT);
-    }
```

Comment:
Am I missing something here? Why are the tests in this file removed?

Reply:
They were deleted when merging with develop, which is very weird. I added them back. Thanks!